### DISCUSSION TITLE
Help to activate the Evaluation Expression Feature

### DISCUSSION THREAD

**Discussion by FurchRadeon** [micronaut-projects/micronaut-core#10107] [Q&A]
Hello, community!

I'm seeking a method to enable the evaluation expression. So far, I've experimented with several ideas, but none have been successful. I've attempted to replicate a project similar to the one outlined in the documentation under the Expression Evaluation Chapter.

Any ideas will may help. Thank you 

I am using Micronaut Version 4.1.6.

[ExpressionEvaluation.zip](https://github.com/micronaut-projects/micronaut-core/files/13327949/ExpressionEvaluation.zip)

**Comment by graemerocher** [comment]
You can't unfortunately mix expression contexts and code in the same source tree. You would need a multi-project build where the `ContextRegistrar` in in a separate sub module and placed in the annotation processor classpath of the consumer project.

**Reply by FurchRadeon** [reply]
Thank you for responding. I will attempt the suggested method.
